src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -175,9 +199,13 @@ public void robotInit() {
    */
   @Override
   public void robotPeriodic() {
-
+    SmartDashboard.putNumber("Time Remaining", DriverStation.getInstance().getMatchTime());//Useful Method to get match data.
     // process();
     Scheduler.getInstance().run();
+
+    SmartDashboard.putNumber("Elevator encoder position", Robot.elevatorSubsystem.getPosition());
```

Comment:
can we capitalize these variable names? nit picky but it looks nicer if we’re going to use SmartDashboard or Shuffleboard as backup